<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_firstpage</class>
 <widget class="QWidget" name="admin_firstpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>صفحه ادمین</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(110, 110, 170)</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>برای حذف یک دکتر ، نام وی را انتخاب و گزینه حذف را فشار دهید.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>حذف</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
align=&quot;center&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>نام : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Post">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>&lt;&lt; مدیریت اخبار و اعلامیه ها &gt;&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Doctor">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>&lt;&lt; مدیریت دکترها &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>تیتر :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>متن:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_subject">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savenews">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>انتشار</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>211</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);

</string>
   </property>
   <property name="text">
    <string>نمایش و ویرایش تمام اعلامیه ها</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
